$NetBSD: patch-ad,v 1.2 2015/12/29 04:22:33 dholland Exp $

Don't sync on Interix. (XXX: shouldn't sync the whole system even not
on Interix. How about using fsync?)

--- osunix/osblock.c.orig	2003-10-20 22:32:26.000000000 -0400
+++ osunix/osblock.c
@@ -293,5 +293,7 @@ void blksync()
 		return;
 #endif
 
+#ifndef __INTERIX
 	sync();
+#endif
 }
